### PR TITLE
chore(litmus-portal): Workflow dashboard layout and a separate route for create workflow

### PR DESCRIPTION
**Pull request by amityt**
## Proposed changes

Added the workflow dashboard layout, workflow table and a separate route for creating a workflow.

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

File: litmus-portal/frontend/src/components/Dashboard/CustomStatus/styles.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    color: theme.palette.secondary.dark,
+    backgroundColor: 'rgba(133, 140, 221, 0.1)',
+  },
+  failed: {
+    width: '3.9125rem',
+    textAlign: 'center',
+    borderRadius: 3,
+    paddingTop: theme.spacing(0.375),
+    paddingBottom: theme.spacing(0.375),
+    color: '#CA2C2C',
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/Dashboard/CustomStatus/styles.ts:28]: Can you put these colours by changing the theme data as I showed you?

File: litmus-portal/frontend/src/components/Dashboard/WorkflowTabs/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  );
+};
+interface StyledTabProps {
+  label: string;
+}
+const StyledTab = withStyles((theme) =>
+  createStyles({
+    root: {
+      textTransform: 'none',
+      color: 'rgba(0,0,0,0.5)',
```
Comment (arkajyotiMukherjee) [inline litmus-portal/frontend/src/components/Dashboard/WorkflowTabs/index.tsx:35]: This too like the function we talked about...you can hit me up if you need help with this

File: litmus-portal/frontend/src/components/Dashboard/WorkflowTabs/index.tsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+const TabPanel: React.FC<TabPanelProps> = ({ children, index, value }) => {
+  return (
+    <div role="tabpanel" hidden={value !== index}>
+      {value === index && (
+        <Box p={3}>
+          <>{children}</>
```
Comment (S-ayanide) [inline litmus-portal/frontend/src/components/Dashboard/WorkflowTabs/index.tsx:22]: Since children is already a React Node do we need fragments here?

**Comment by S-ayanide** [comment]
@AmitKumarDas CI is failing at  LinearProgressBar.spec.tsx Unit Test, can you take a look at that?

**Comment by amityt** [comment]
> @amityt CI is failing at LinearProgressBar.spec.tsx Unit Test, can you take a look at that?

Yes @S-ayanide , I'll take a look.Added some changes in the themes file yesterday. Will update the LinearProgresBar unit test.

File: litmus-portal/frontend/cypress/component/LinearProgressBar.spec.tsx [hunk 4]
```diff
@@ -4,20 +4,36 @@ import { mount } from 'cypress-react-unit-test';
 import LinearProgressBar from '../../src/components/ProgressBar/LinearProgressBar';
 
 describe('Linear Progressbar Testing', () => {
-  it('Progressbar color is Red for value 3', () => {
+  it('Progressbar stroke for value 3', () => {
     mount(<LinearProgressBar value={2} />);
```
Comment (Jonsy13) [inline litmus-portal/frontend/cypress/component/LinearProgressBar.spec.tsx:8]: In it(), you have given that you are testing for value 3, but during mounting the component,  the value given is 2?

Comment (amityt) [reply litmus-portal/frontend/cypress/component/LinearProgressBar.spec.tsx:8]: Thanks for the catch @Jonsy13 . I'll make the changes!

**Pull request merged**